src/clamp.js

```diff
@@ -0,0 +1,29 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Restricts a number to be within a range.
+ *
+ * Also works for other ordered types such as Strings and Dates.  (see Type::Ord)
```

Comment:
I don't think @CrossEye was suggesting to actually include the parenthesized text. I believe he was saying it would be nice if we could do something like this and have it link to an explanation of Ord.